app/controllers/concerns/idv/outage_concern.rb

```diff
@@ -11,10 +11,6 @@ def check_for_outage
     def redirect_for_gpo_only
       return redirect_to vendor_outage_url unless FeatureManagement.gpo_verification_enabled?
 
-      # During a phone outage, skip the hybrid handoff
```

Comment:
This logic has moved to HybridHandoffController, which will redirect straight to document capture if the hybrid flow is unavailable.